freetextresponse/tests.py

```diff
@@ -260,6 +261,23 @@ def test_compute_score(self, credit):
             {'value': credit.value, 'max_value': Credit.full.value},
         )
 
+    @ddt.data(Credit.zero, Credit.half, Credit.full)
+    def test_compute_score_error(self, credit):
```

Comment:
Error is pretty vague.  Let's make the function name more specific, like  test_compute_score_integrity_error